src/components/Switch.jsx

```diff
@@ -46,13 +48,27 @@ module.exports = React.createClass({
                        onExit={ this.close } />)
   },
 
+  hasMaxChildren() {
+    let { app, parent } = this.props
+
+    if (!parent) return false
```

Comment:
Right now no. This is sort of a design flaw in CK right now... I'll have to add it as a separate feature.

I'm starting to think that the root level editor for Colonel Kurtz should be a block itself. Then you could add the same options to blocks as the editor. Right now, that's not the case :-/